lesspipe.sh.in

```diff
@@ -160,7 +160,7 @@ filetype () {
        # Compressed Archives
   elif [[ "$type" != *lzip\ compressed* && ("$name" = *.lzma || "$name" = *.tlz) ]]; then
     return=" LZMA compressed data"
-  elif [[ ("$type" = *Zip* || "$type" = *ZIP*) && ("$name" = *.jar || "$name" = *.xpi) ]]; then
+  elif [[ ("$type" = *Zip* || "$type" = *ZIP* || "$type" = *JAR*)) && ("$name" = *.jar || "$name" = *.xpi) ]]; then
```

Comment:
The extra closing parenthesis causes a syntax error.

Reply:
Thanks for pointing to this. As this seems to be a bug-report (or even a patch): Please submit it as such. Otherwise it will be buried here. (Of course, I could do, but the honor is yours!)